docs/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM python
+
+COPY --chown=1000:1000 ./requirements.txt ./
+COPY --chown=1000:1000 ./run-documentation.sh ./
+
+# RUN chmod +x ./run-documentation.sh
+
+RUN pip install mkdocs
+RUN pip install mkdocstrings
+RUN pip install mkdocs-material
+
+RUN export PYTHONPATH=/wazuh-qa/deps/wazuh_testing
```

```suggestion
ENV PYTHONPATH=/wazuh-qa/deps/wazuh_testing
```
if you declare an env variable with this, it will be global to the image and you won't need to export it later.

---

Done in 618a8892f93dd794a6000baa838503371ec1eb6d